webdriver-spec.html

```diff
@@ -6622,10 +6622,10 @@
 
  <!-- TODO: consider picking the current input of the right
   type if one exists -->
- <li><p>If <var>id</var> is <a>undefined</a> let <var>id</var> be a
-  new <a>UUID</a>. Otherwise if <var>id</var> is not a <a>String</a>
-  return <a>error</a> with <a>error code</a> <a>invalid
-  argument</a>.
+ <li><p>If <var>id</var> is <a>undefined</a> return an <a>error</a>
+  with <a>error code</a> <a>invalid argument</a>. Otherwise
```

Comment:
I think this can be simplified to ` is <a>undefined</a> or is not a <a>String</a> return <a>error</a> with <a>error code</a> <a>invalid argument</a>.`